### PR TITLE
Add Person Gallery Vertical

### PR DESCRIPTION
**Pull request by giansalvo**
I added the new class PersonGalleryVertical which inherit from PersonGallery and the new gramplet Person Gallery Vertical to show media object vertically. If this idea is accepted I will do the same for the other galleries (Places, Families, etc)

See bug [#11527](https://gramps-project.org/bugs/view.php?id=11527).

### CODE REVIEW

File: gramps/plugins/gramplet/gallery.py [hunk 1]
```diff
@@ -49,7 +51,10 @@ def build_gui(self):
         Build the GUI interface.
         """
         self.image_list = []
-        self.top = Gtk.Box(spacing=3)
+        if self.direction == "Horizontal":
+            self.top = Gtk.Box(spacing=3)
+        else:
+            self.top = Gtk.VBox(spacing=3)
```
Comment (Nick-Hall) [inline gramps/plugins/gramplet/gallery.py]: This has been deprecated.  Use `Gtk.Box(spacing=3, orientation=Gtk.Orientation.VERTICAL)` instead.

**Comment by Nick-Hall** [comment]
Thanks for your contribution, however, I'm not sure that creating a new plugin is the best way to solve this issue.  I'll investigate and come up with a suggestion.

**Comment by giansalvo** [comment]
> 
> 
> Thanks for your contribution, however, I'm not sure that creating a new plugin is the best way to solve this issue. I'll investigate and come up with a suggestion.

@Nick-Hall  Thanks for your feedback. Another way would be to make the gramplet differentiate if it is loaded in the sidebar. This would require to maybe change the opts structure of the gramplets. I don't know if this is a good way either. Let me know your directions and I'll be happy to implement the changes.

**Comment by prculley** [comment]
Another idea would be to consider using a Gtk.IconView.  I think this would work in either orientation, filling the available space the way file managers display icons.

**Comment by Nick-Hall** [comment]
Good idea.  This is certainly worth trying for the gallery gramplets.  Other gramplets may benefit from a horizontal and vertical format, but we can think about that later.

**Comment by Nick-Hall** [comment]
Replaced by PR #1370.

**Pull request closed without merging**